pkg/kclient/binding.go

```diff
@@ -0,0 +1,142 @@
+package kclient
+
+import (
+	"context"
+	"errors"
+
+	sboApi "github.com/redhat-developer/service-binding-operator/apis/binding/v1alpha1"
+	kerrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/api/meta"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+const (
+	ServiceBindingGroup    = "binding.operators.coreos.com"
+	ServiceBindingVersion  = "v1alpha1"
+	ServiceBindingKind     = "ServiceBinding"
+	ServiceBindingResource = "servicebindings"
+	BindableKindsResource  = "bindablekinds"
+)
```

Comment:
You can use the constants defined in https://github.com/redhat-developer/service-binding-operator/blob/master/apis/binding/v1alpha1/groupversion_info.go